uniffi_bindgen/src/bindings/kotlin/templates/UByteArrayHelper.kt

```diff
@@ -0,0 +1,16 @@
+@OptIn(kotlin.ExperimentalUnsignedTypes::class)
+public object FfiConverterUByteArray: FfiConverterRustBuffer<UByteArray> {
```

Comment:
Thanks so much for putting this PR together!

How would you (and other folks reviewing the PR) feel about using `ByteArray` instead of `UByteArray` in Kotlin?

It looks like it's possible to use `UByteArray.asByteArray()` to turn a `UByteArray` into a `ByteArray` without an extra copy, but many Java and Android interfaces work with `ByteArray` exclusively, and it might be nice to avoid the `toUByteArray`/`toByteArray` copies in `read`/`write`, and also avoid having to opt-in to experimental unsigned types every time...

Reply:
> and it might be nice to avoid the `toUByteArray`/`toByteArray` copies in `read`/`write`

I didn't see the `UByteArray.asByteArray` function when composing the PR, it seems there's also the inverse `ByteArray.asUByteArray`, so this could all be done without copies, and the performance point is moot.

> How would you (and other folks reviewing the PR) feel about using `ByteArray` instead of `UByteArray` in Kotlin?

I don't know enough about Kotlin to have feelings about this, really. I went with `UByteArray` because it's unsigned in the other languages, too, but I'll happily change it to `ByteArray` if that's more in the way of Kotlin.

Reply:
Done.